robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -25,175 +11,84 @@ const uint8_t RX_TEENSY_3_6_PIN = 0;
 }
 
-
-void attachMotors(){
-
-    Motor::attachMotor(FRONT_RIGHT,M1_FR_DIR,M1_FR_PWM,GEAR_RATIO);
-    Motor::attachMotor(MIDDLE_RIGHT,M2_MR_DIR,M2_MR_PWM,GEAR_RATIO);
-    Motor::attachMotor(REAR_RIGHT,M3_RR_DIR,M3_RR_PWM,GEAR_RATIO);
-
-    Motor::attachMotor(FRONT_LEFT,M4_FL_DIR,M4_FL_PWM,GEAR_RATIO);
-    Motor::attachMotor(MIDDLE_LEFT,M5_ML_DIR,M5_ML_PWM,GEAR_RATIO);
-    Motor::attachMotor(REAR_LEFT,M6_RL_DIR,M6_RL_PWM,GEAR_RATIO);
-}
-void initPidControllers(){
-
-    Motor::initPidController(FRONT_RIGHT,14.1,0.282,40.625);
-    Motor::initPidController(MIDDLE_RIGHT,14.1,0.282,40.625);
-    Motor::initPidController(REAR_RIGHT,14.1,0.282,40.625);
-
-    Motor::initPidController(FRONT_LEFT,14.1,0.282,40.625);
-    Motor::initPidController(MIDDLE_LEFT,14.1,0.282,40.625);
-    Motor::initPidController(REAR_LEFT,14.1,0.282,40.625);
-}
-void attachEncoders(){
-
-    Motor::attachEncoder(FRONT_RIGHT,M1_FR_A,M1_FR_B,PULSES_PER_REV,InterruptHandler::RightFrontMotorInterruptHandler);
-    Motor::attachEncoder(MIDDLE_RIGHT,M2_MR_A,M2_MR_B,PULSES_PER_REV,InterruptHandler::RightMiddleMotorInterruptHandler);
-    Motor::attachEncoder(REAR_RIGHT,M3_RR_A,M3_RR_B,PULSES_PER_REV,InterruptHandler::RightBackMotorInterruptHandler);
-
-    Motor::attachEncoder(FRONT_LEFT,M4_FL_A,M4_FL_B,PULSES_PER_REV,InterruptHandler::LeftFrontMotorInterruptHandler);
-    Motor::attachEncoder(MIDDLE_LEFT,M5_ML_A,M5_ML_B,PULSES_PER_REV,InterruptHandler::LeftMiddleMotorInterruptHandler);
-    Motor::attachEncoder(REAR_LEFT,M6_RL_A,M6_RL_B,PULSES_PER_REV,InterruptHandler::LeftBackMotorInterruptHandler);
-}
 void attachServos(){
-    Rover::attachServo(FRONT_BASE_SERVO,FB_SERVO);
-    Rover::attachServo(FRONT_SIDE_SERVO,FS_SERVO);
-    Rover::attachServo(REAR_SIDE_SERVO,RS_SERVO);
-    Rover::attachServo(REAR_BASE_SERVO,RB_SERVO);
-}
-void writeServoDefaultValues(){
-    Rover::writeToServo(FRONT_BASE_SERVO,FRONT_BASE_DEFAULT_PWM);
-    Rover::writeToServo(FRONT_SIDE_SERVO,SERVO_STOP);
-    Rover::writeToServo(REAR_BASE_SERVO,REAR_BASE_DEFAULT_PWM);
-    Rover::writeToServo(REAR_SIDE_SERVO,SERVO_STOP);
+    Rover::attachServo(CENTER_BACK_1_SERVO,CB_1_SERVO);
+    Rover::attachServo(CENTER_BACK_2_SERVO,CB_2_SERVO);
 }
 
+void attachMotors(){
+    Motor::attachMotor(FRONT_RIGHT,M1_FR_DIR,M1_FR_PWM);
+    Motor::attachMotor(MIDDLE_RIGHT,M2_MR_DIR,M2_MR_PWM);
+    Motor::attachMotor(REAR_RIGHT,M3_RR_DIR,M3_RR_PWM);
 
-void WheelsCommandCenter::enableMotors(uint8_t turnMotorOn) {
-    //rover->enableAllMotors( (bool) turnMotorOn);
+    Motor::attachMotor(FRONT_LEFT,M4_FL_DIR,M4_FL_PWM);
+    Motor::attachMotor(MIDDLE_LEFT,M5_ML_DIR,M5_ML_PWM);
+    Motor::attachMotor(REAR_LEFT,M6_RL_DIR,M6_RL_PWM);
 }
-
 void WheelsCommandCenter::stopMotors() {
     Rover::stopMotors();
-    Rover::systemStatus.last_throttle = millis();
 }
 
-void WheelsCommandCenter::closeMotorsLoop() {
-    Rover::closeLoop();
+void WheelsCommandCenter::moveRover(const float & linear_y,const float & omega_z) {
```

Comment:
Did you get this working in the end?

Reply:
need to test it out a bit again(this is for twist commands)

Reply:
like i think the direction needs to be reversed and the sensitivity needs to be adjusted iirc